eoscostarica/evodex
# feat: add timeout for ual api failover

### GH Issue #238

### What does this PR do?
Add a timeout to use the failover api

### Steps to test
1. yarn
2. yarn start
3. navigate to http://localhost:3000/exchange
4. meanwhile the ual api its resolved will display a "Loading..." message
5. the "Loading..." message should gone away after the time set in `REACT_APP_EOS_API_TIMEOUT` env variable 

#### CheckList
